beacon-chain/sync/rpc_data_column_sidecars_by_root.go

```diff
@@ -69,25 +78,50 @@ func (s *Service) dataColumnSidecarByRootRPCHandler(ctx context.Context, msg int
 	}
 
 	custodiedColumns, err := peerdas.CustodyColumns(s.cfg.p2p.NodeID(), custodiedSubnets)
-
 	if err != nil {
 		log.WithError(err).Errorf("unexpected error retrieving the node id")
 		s.writeErrorResponseToStream(responseCodeServerError, types.ErrGeneric.Error(), stream)
-		return err
+		return errors.Wrap(err, "custody columns")
 	}
 
-	for i := range columnIdents {
+	custodiedColumsLists := make([]uint64, 0, len(custodiedColumns))
```

Comment:
```suggestion
	custodiedColumnsList := make([]uint64, 0, len(custodiedColumns))
```

Reply:
Fixed in cdbae5e2dd652d1190968244c8826624afae496e.

(I did not apply you commit suggestion since it would break the build.)

Reply:
@nalepae looks like in current fix you are still missing the n in `Columns`

Reply:
Re-fixed in 8f60e357e9fc0017093e32e711815a81eac33e03.